data/campaigns/Dead_Water/scenarios/10_The_Flaming_Sword.cfg

```diff
@@ -1068,7 +1068,7 @@
             [then]
                 [message]
                     speaker=Kai Krellis
-                    message= _ "We thank all you humans for your help, and release you from any farther obligations. Where we are going, you cannot easily follow."
+                    message= _ "We thank all you humans for your help, and release you from any further obligations. Where we are going, you cannot easily follow."
```

Comment:
[Further vs Farther](https://www.grammarly.com/blog/farther-further/#:~:text=People%20use%20both%20further%20and,and%20further%20for%20figurative%20distances.)

Not sure about this one. Perhaps a native English Speaker might add in their opinion?

Reply:
yeah "further" is better

Reply:
> So, general comment: Kai Krellis is royalty; if his language seems intentionally heightened/archaic, then that's probably intentional

There are only a small countable number of changes, would be helpful if you could reference specific edits.

Reply:
The convention matches the article you have linked. We are dealing with a figurative distance in this sentence, not a physical distance.

Reply:
This change is definitely correct.